deisctl/cmd/cmd.go

```diff
@@ -405,7 +405,7 @@ func Config(target string, action string, key []string) error {
 // RefreshUnits overwrites local unit files with those requested.
 // Downloading from the Deis project GitHub URL by tag or SHA is the only mechanism
 // currently supported.
-func RefreshUnits(dir string, tag string) error {
+func RefreshUnits(dir string, tag string, get func(string) (*http.Response, error)) error {
```

Comment:
i.e. make url a parameter, and when we call RefreshUnits, we send in that URL. In the default case, that'd be the github URL and for the httptest server that'd be `srv.URL`.